pyproject.toml

```diff
@@ -52,13 +53,13 @@ prefixcommons = ">=0.1.12"
 pyyaml = "*"
 rdflib = ">=6.0.0"
 requests = "*"
-prefixmaps = ">=0.1.4"
+prefixmaps = ">=0.2.2"
 curies = ">=0.5.4"
 pydantic = ">=1.10.2, <3.0.0"
 
 [tool.poetry.dev-dependencies]
 coverage = "^6.2"
-importlib-metadata = {version = "^4.12.0", python = "<3.8"}
+importlib-metadata = {version = "^4.12.0", python = "^3.8"}
```

Comment:
i think this can get removed? or else if we depend on features specifically from `^4.12.0` it looks like we can do `python = "^3.10"`

Reply:
(also lgtm but idk if i'm allowed to give you a thumbs up review lol)

Reply:
what i would say is that for the `metadata()` call to get the version here, which is the one place that it's used: https://github.com/linkml/linkml/blob/7ea1a33226bb6480c7e0bea5b1c5ad11347dcceb/linkml/_version.py#L5

if we are going `>=3.8` then we can remove it, for its purpose has been served <3

it is useful in the sense that it forced me to implement a whole bunch of specific behavior for conditional imports i wouldn't have remembered to otherwise (but will inevitably be needed at some point) in this: https://github.com/linkml/linkml/pull/1927/files

so it gave us one last gift (and hints that we should probably take that out of the python generators too

Reply:
`importlib-metadata` can then be completely from `[tool.poetry.dev-dependencies]`, right?

Reply:
ya and nab the place it's imported in `_version` and should be good?

Reply:
For some reason I am not seeing "resolve conversation" on my end, so if you do, then resolve it :)